RCSwitch.h

```diff
@@ -48,6 +48,13 @@
 // We can handle up to (unsigned long) => 32 bit * 2 H/L changes per bit + 2 for sync
 #define RCSWITCH_MAX_CHANGES 67
 
+#define RCSWITCH_PROTOCOL_1 1
+#define RCSWITCH_PROTOCOL_2 2
+#define RCSWITCH_PROTOCOL_3 3
+#define RCSWITCH_PROTOCOL_4 4
+#define RCSWITCH_PROTOCOL_5 5
+#define RCSWITCH_HT6P20B 6
+
```

Comment:
These new #defines are also not mentioned in the description of the pull request.

I guess it makes sense to have symbolic names for the predefines protocols, and encourage people to use them. But then this should be done by defininig an enum, and `setProtocol` could be changed to take such an enum value as argument.